assets/js/state/databases.js

```diff
@@ -116,6 +139,8 @@ export const {
   updateDatabaseInstanceSystemReplication,
   addTagToDatabase,
   removeTagFromDatabase,
+  setDatabaseInstanceDeregistering,
+  setDatabaseInstanceNotDeregistering,
```

Comment:
Not 100% sure that I understand the reasoning behind the naming properly, but wouldn't something like `unsetDatabaseInstanceDeregistering` be more accurate?